tests/app/billing/test_billing.py

```diff
@@ -282,8 +282,8 @@ def test_create_free_sms_fragment_limit(client, sample_service):
     json_resp = json.loads(response_get.get_data(as_text=True))
     assert response.status_code == 201
     assert response_get.status_code == 200
-    assert json_resp['data']['financial_year_start'] == 2017
-    assert json_resp['data']['free_sms_fragment_limit'] == 250
+    assert json_resp['financial_year_start'] == 2017
```

Comment:
Need to add in the `['data']` block if you choose to return the json block in `['data']`, you could put this block as part of the `json_resp` setting in line 282 rather than adding it here. 
You will also need to refactor the other tests.

Reply:
This should be OK as long as consistence between front and backend